scripts/release/rpm/test_azurelinux_in_docker.sh

```diff
@@ -14,7 +14,7 @@ time az self-test
 time az --version
 
 cd /azure-cli/
-python -m pip install setuptools
+python -m pip install --upgrade setuptools
```

Comment:
My comment https://github.com/Azure/azure-cli/pull/30211#discussion_r1820174098 is valid then. 😉